src/components/examples-tools/example-event-printer/example-event-printer.tsx

```diff
@@ -0,0 +1,124 @@
+import { Component, h, Method, Prop, State } from '@stencil/core';
+import dayjs from 'dayjs';
+
+/**
+ * This component is only used in our documentations
+ * to render caught events from the components that are showcased
+ * in the examples.
+ *
+ * @exampleComponent kompendium-example-example-event-printer
+ */
+@Component({
+    tag: 'kompendium-example-event-printer',
```

Comment:
I think we should rename this and make the component a little bit more generic. Instead of only allowing events to be printed, I think we could allow anything that is serializable

Reply:
I don't understand. 😓

Reply:
I disagree. I think the whole point of this component is that we have this little formatter:

<img width="529" alt="image" src="https://user-images.githubusercontent.com/154725/215485657-beee5dab-b0e9-41a6-bfe1-9e0bf0e4c1b0.png">

<img width="501" alt="image" src="https://user-images.githubusercontent.com/154725/215486386-526526d1-3f9a-43c1-99a4-3c4d0e301dc6.png">

Without that, I don't think there's any reason for this component to exist at all… 😕